lib/web_ui/lib/src/engine/html/recording_canvas.dart

```diff
@@ -51,7 +51,7 @@ class RawRecordingCanvas extends BitmapCanvas implements ui.PictureRecorder {
 
   late RecordingCanvas _canvas; // ignore: unused_field
 
-  bool _isRecording = true; // ignore: unused_field
+  final bool _isRecording = true; // ignore: unused_field
```

Comment:
Looks like `RawRecordingCanvas` class is used anywhere. I think it's dead code.

Reply:
Deleted.